contrib/scripts/mint_nbtc.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+PACKAGE_ID="0x5419f6e223f18a9141e91a42286f2783eee27bf2667422c2100afc7b2296731b"
+TREASURY_CAP_ID="0x47336d196275369fb52a200682a865a4bffdc9469d755d418d7e985c376ace35"
+LIGHT_CLIENT_ID="0x4f989d395bb13b4913b483016641eb7c9cacfd88d2a1ba91523d0542a52af9e4"
+
+# Bitcoin Transaction Details
+BTC_TX_VERSION="0x02000000"
```

Comment:
You can extract nBTC mint arguments from create_btc_mint_data.sh. 
For example: 
```bash
mint_data = $(./create_btc_mint_data.sh) 
BTC_TX_VERSION=mint_data| jq ".version"
BTC_SERIALIZED_INPUTS=mint_data | jq ".inputs"
```